src/channel.ts

```diff
@@ -1391,6 +1391,24 @@ export class Channel<StreamChatGenerics extends ExtendableGenerics = DefaultGene
           channelState.clearMessages();
         }
         break;
+      case 'user.banned':
+        if (!event.user?.id) break;
+        channelState.members[event.user.id] = {
```

Comment:
We should check whether the user banned/unbanned is a member of the channel before adding to `channelState.members`

Reply:
Ah, ok, I can see that there are two kinds of bans:
- `channel.banUser()`
- `client.banUser()`

I can see now that this feature is for `channel.banUser`